ramls/request.json

```diff
@@ -133,6 +133,22 @@
         "barcode": {
           "description": "barcode of the item",
           "type": "string"
+        },
+        "itemEffectiveLocationId": {
+          "description": "Item's effective location",
+          "type": "string"
```

Comment:
Should there be a pattern or $ref attributes for the ID's ? for e.g. "$ref": "raml-util/schemas/uuid.schema"

Reply:
fixed:=> https://github.com/folio-org/mod-circulation-storage/pull/498/commits/9a80571bd56b1a2d0f308c1bfb41dffdaf162896